pkg/agent/facts/facts.go

```diff
@@ -34,12 +38,28 @@ func (col factCollector) fetchDynamicFacts(ctx context.Context) (*api.DynamicClu
 		facts["openshiftVersion"] = ocpVersion
 	}
 
+	ocpOAuthRoute, err := col.fetchOpenshiftOAuthRoute(ctx)
+	if err != nil {
+		klog.Errorf("Error fetching openshift oauth route: %v", err)
+	}
+	if ocpOAuthRoute != "" {
+		facts["openshiftOAuthRoute"] = ocpOAuthRoute
+	}
+
+	additionalFacts, err := col.fetchAdditionalFacts(ctx)
+	if err != nil {
+		klog.Errorf("Error fetching additional facts: %v", err)
+	}
+	for k, v := range additionalFacts {
+		facts[k] = v
+	}
```

Comment:
I guess it doesn't really make a difference, but there's an argument for not allowing the configmap to override "real" dynamic facts.

It won't matter 99% of the time in practice, but it might lead to confusion if someone accidentally puts a K8s version fact in the configmap.

Reply:
True, I saw it as a feature to override faulty facts but there's also arguments to have it the other way around.